chain/epoch-manager/src/lib.rs

```diff
@@ -1101,7 +1099,7 @@ impl EpochManager {
         let chunk_producers_settlement = epoch_info.chunk_producers_settlement();
         let chunk_producers = chunk_producers_settlement
             .get(shard_index)
-            .ok_or_else(|| EpochError::ShardingError(format!("invalid shard id {shard_id}")))?;
+            .ok_or(EpochError::ShardingError(format!("invalid shard id {shard_id}")))?;
```

Comment:
This is less efficient right? We will format the string and construct the error even if it's not needed. But no big deal since it's just the constructor for epoch manager (runs once).

Reply:
I think clippy would complain if we don't use `ok_or_else`